app/actions/remote/notifications.ts

```diff
@@ -119,6 +119,36 @@ export const backgroundNotification = async (serverUrl: string, notification: No
 
             if (posts) {
                 const postsData = processPostsFetched(posts);
+
+                // We need to convert the following properties from 1/0 to true/false, because
+                // we validate message attachments before displaying them.
+                // This is a problem from the NSJSONSerialization call in AppDelegate.mm -- there
+                // is no true/false in Objective-C, only NSNumber with value 1 or 0
+
+                // Convert attachment.fields.short, and attachment.actions.disabled
+                postsData.posts.forEach((post) => {
```

Comment:
0/5 but since this is only a problem in objective c, should we add here a check to only run this logic on iOS?

Reply:
Yep, good call, thanks!